openerp-atomfeed-service/src/test/java/org/bahmni/feed/openerp/event/OpenERPCustomerServiceEventWorkerTest.java

```diff
@@ -38,7 +40,7 @@ public void setUp() throws Exception {
     public void shouldCallOpenERPClientWithRightParameters() throws FileNotFoundException {
         MRSURLPrefix = "urlPrefixTest";
         OpenERPCustomerServiceEventWorker customerServiceEventWorker =
-                new OpenERPCustomerServiceEventWorker("www.openmrs.com", openERPContext, mockWebClient, MRSURLPrefix);
+                new OpenERPCustomerServiceEventWorker("www.openmrs.com", endpointURI, openERPContext, mockWebClient, MRSURLPrefix);
```

Comment:
Why do we pass `www.openmrs.com`? This is not even a valid domain. OpenMRS domain is openmrs.org.

Reply:
No network calls are being made here and we just test if a request gets created with the given payload and endpoints.